inference-engine/src/vpu/graph_transformer/src/stages/proposal.cpp

```diff
@@ -128,8 +131,15 @@ void FrontEnd::parseProposal(const Model& model, const ie::CNNLayerPtr& layer, c
     // #-37327: Several models Failed to compile layer "proposals"
```

Comment:
Please, update this comment. I guess, it is resolved by this PR

Reply:
done